### ISSUE TITLE
OpenPGP encryption for email notifications

### ISSUE THREAD

**Issue by qntmpkts** [keybase/keybase-issues#2753]
When receiving email notifications, if a user has provided a public OpenPGP formatted public key associated with an email account, emails addressed to that account should be encrypted using their public key and formatted accordingly.

For instance, Facebook added OpenPGPjs support to allow ProtonMail users to receive emails encrypted with their ProtonMail public key (tied to the ProtonMail email account of their choosing): cf. https://protonmail.com/blog/protonmail-facebook-pgp . In this particular example, the PGP format Facebook follows is OpenPGPjs compatible. For more information by Facebook: https://www.facebook.com/notes/protecting-the-graph/securing-email-communications-from-facebook/1611941762379302 .

Although at this time this would mostly benefit ProtonMail users, I suspect flexibly extending this to other formats might be a boon to other users.

**Comment by mkosmo** [comment]
I'd also love to have this function.  The chat notifications got me thinking about it most:  I really don't like having a plaintext email pointing to the fact that I have pending encrypted communications pending.

**Comment by jvoss** [comment]
I tend to agree with mkosmo. It would make more sense to PGP encrypt email notifications.

**Comment by mrBussy** [comment]
I'd love to have this feature being developed. I recently started my interest again in Keybase and I would really love this feature since it makes it even more secure.
